gdp-src-build/rokers-developer.conf

```diff
@@ -2,6 +2,10 @@
 
 BBLAYERS += " \
   ${TOPDIR}/../meta-rokers \
+  ${TOPDIR}/../openembedded-core/meta \
+  ${TOPDIR}/../meta-openembedded/meta-oe \
+  ${TOPDIR}/../meta-openembedded/meta-python \
```

Comment:
Those three layers are not properly spelled, and anyway are already included by

* https://github.com/robotrokers/genivi-dev-platform/blob/dev-rokers/gdp-src-build/conf/templates/bblayers.inc#L9
* https://github.com/robotrokers/genivi-dev-platform/blob/dev-rokers/gdp-src-build/conf/templates/bblayers.inc#L14
* https://github.com/robotrokers/genivi-dev-platform/blob/dev-rokers/gdp-src-build/conf/templates/bblayers.inc#L18